src/wasm.h

```diff
@@ -916,6 +916,8 @@ class Module {
 
   std::vector<UserSection> userSections;
   std::vector<std::string> debugInfoFileNames;
+  FeatureSet features = FeatureSet::MVP;
+  bool hasFeatures = false;
```

Comment:
this worries me - we'll have features on both the module and the pass options. the latter feels like the only right place to me?

Reply:
I agree. I'll refactor to have `readFeatures` return the features directly to the caller instead of storing them on the module.